frontend/src/components/abfragen/AbfrageComponent.vue

```diff
@@ -214,6 +254,29 @@ export default class AbfrageComponent extends Mixins(
     return this.$store.getters["lookup/statusAbfrage"];
   }
 
+  private isCancelled(): boolean {
+    return this.abfrage.statusAbfrage === StatusAbfrage.Abbruch;
+  }
+
+  private getStatusIndex(): number {
+    switch (this.abfrage.statusAbfrage) {
+      case StatusAbfrage.Angelegt:
```

Comment:
Wird der Status "Abbruch" nicht berücksichtigt?

![image](https://github.com/it-at-m/isi-frontend/assets/56300801/b00366c2-eb5b-400a-8495-bf2452f2133a)

Reply:
Das wird im HTML in Zeile 28-33 abgefangen. Wir waren der Ansicht, dass "Abbruch" ein Spezialfall ist, der sich schlecht in den normalen Verlauf bei der Status-Leiste einordnen lässt. In der Story gabs dazu auch keine Vorgaben.